Cargo.lock

```diff
@@ -4,19 +4,19 @@ version = 3
 
 [[package]]
 name = "actix-codec"
-version = "0.5.0"
```

Comment:
there are a lot of unrelated dependency changes here, did you run cargo update on the all repo?

Reply:
I did 😬